### ISSUE TITLE
Rework the raw profile JSON data comparison check for less trouble

### ISSUE THREAD

**Issue by Max-Rodriguez** [Max-Rodriguez/Instagram-Offline#8]
The first comparison made (the raw profile JSON data comparison) keeps detecting false positives, with more and more certain parts of the JSON that are dynamic. I've been making simple regex + .replace() fixes to strip out dynamic data that should be ignored when comparing, but I think it will be a smarter idea to rework this first comparison so that we can also output specific detected changes from the profile, and more easily ignore data that doesn't need to be compared, instead of stringifying the entire fetched and archive JSON documents.

In summary:
    - Rework the profile raw JSON data comparison, with a solution that avoids comparing JSONs via `JSON.stringify()`.
    - Allow for more specific logging on specific detected profile changes, rather than just comparing the JSON strings as a whole.
